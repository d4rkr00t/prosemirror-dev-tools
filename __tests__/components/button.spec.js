import React from "react";
import renderer from "react-test-renderer";

var Button = require("../../src/components/button").default;
var input = [
  {
    isActive: "1",
    theme: "white05"
  }
];

test("Button renders when active", () => {
  var button = renderer.create(<Button params={input} />).toJSON();
  expect(button).toMatchSnapshot();
});
