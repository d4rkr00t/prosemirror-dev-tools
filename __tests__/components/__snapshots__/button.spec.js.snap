// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders when active 1`] = `
<button
  className="sc-bdVaJa kKiekB"
/>
`;
